adr/0250-developing-and-publishing-clientside-behaviours.md

```diff
@@ -0,0 +1,48 @@
+# 0250. Developing and Publishing ClientSide Behaviours
+
+Date: 2021-02-19
+
+## Status
+
+Proposed
+
```

Comment:
What do you think about only merging accepted ADRs? Then we could remove this field.

```suggestion
## Status

Proposed

```

Reply:
The field is useful for `Superseded`/`Deprecated` decisions. I also feel like there are good motivations for intentionally documenting an ADR as `Rejected` - to document decisions which we don't want to implement (with reasons as to why).

Reply:
```suggestion
## Status

Accepted

```